### PR TITLE
upgraded a bit

### PR DESCRIPTION
**Pull request by marvil6**
Now, whatkey can detect multiple keys at once if they are being pressed. For example, Ctrl+Shift+B will be shown if they are pressed. Also made the website dark and added a favicon.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
+            
+            document.onkeydown = function (e) {
+                return false;
+            } // prevent the user from using the keyboard, so the listener is called instead of whatever keybinds they have.
+        </script>
+        <style>
+            .indicator {
+            font-size: 4em;
+            color: #bed5ff;
+            }
```
Comment (marvil6) [inline index.html:27]: Since it's quite small, the JavaScript is now inside index.html

File: index.html [hunk 2]
```diff
@@ -1,20 +1,57 @@
...
 
+            .footer {
+                position: fixed;
+                font-size: 12px;
+                left: 0;
+                bottom: 0;
+                width: 100%;
+                text-align: center;
+            }.footer a { color: #bed5ff; font-size: 2em; } /* Changed footer font size and color */
+        </style>
```
Comment (marvil6) [inline index.html:46]: Same goes for CSS, it's added inside index.html since it's small.

File: index.html [hunk 3]
```diff
@@ -1,20 +1,57 @@
...
+    </head>
+
+    <body>
+        <div id="instructions" class="indicator">Press a key.</div>
+        <div id="pressed" class="indicator"></div>
+        <footer class="footer">
+            <p><a href="https://github.com/nadavoosh/whatkey">feedback welcome</a></p>
+        </footer>
+    </body>
```
Comment (marvil6) [inline index.html:55]: made the footer look a bit better

**Comment by nadavoosh** [comment]
Thanks for the PR @marvil6 !

Lots of different, unrelated changes in here! I love the multiple-key change. Would you be able to pull that into it's own separate PR so I can just review that functionality change? Thank you!

**Pull request closed without merging**